### ISSUE TITLE
Regarding the Accessibility overlay 'AccessiBe'

### ISSUE THREAD

**Issue by nithyaasworld** [topcoder-platform/community-app#5738]
**Describe the bug**
This is regarding the Accessibility overlay 'accessiBe' 

**To Reproduce**
1. Go to www.topcoder.com
2. Check the 'Accessibility overlay' icon present at the left lower corner

**Expected behavior**
This is originally reported here: https://github.com/topcoder-platform/Training-Challenges/issues/279#issuecomment-939038450

Statement by the reporter: 
I would like to add on this, it would really be better to get rid of the Accessibility overlay. They are highly frowned upon in the Accessibility community. 

They cause a lot of problems, do not fix underlying bad code, and any fixes implemented by the overlay are generally not considered WCAG compliant. In fact, since overlays work by altering the underlying code of the site, they can result in greater accessibility issues than those they "solve." 

AccessiBe, the overlay used on TopCoder, is one of the top ones I see experts in Accessibility frequently complain about. It is better to instead implement options in code that a user can set (i.e. high contrast mode, enlarge text, etc.).

Supporting articles:
- [The Many Pitfalls of Accessibility Overlays](https://www.essentialaccessibility.com/blog/the-many-pitfalls-of-accessibility-overlays)
- [A guide to accessibility overlays](https://siteimprove.com/en-us/accessibility/accessibility-overlays/)
- [Why Accessibility Overlay Solutions Fail to Protect or Serve](https://www.accessibility.works/blog/avoid-accessibility-overlay-tools-toolbar-plugins/)
- [Honor the ADA: Avoid Web Accessibility Quick-Fix Overlays](https://www.lflegal.com/2020/08/quick-fix/)
- [Overlay Fact Sheet](https://overlayfactsheet.com/)

**Screenshots**
![image](https://user-images.githubusercontent.com/5114190/136731257-a3b87c91-4cbe-4f98-ab59-f0fc2ecf06e8.png)

**Desktop (please complete the following information):**
 - OS: Mac, Windows
 - Browser: Chrome
